pyproject.toml

```diff
@@ -2,32 +2,28 @@
 name = "petdeface"
 version = "0.0.0"
 description = "A nipype PET and MR defacing pipeline for BIDS datasets utilizing FreeSurfer's MiDeFace."
-authors = ["Martin Nørgaard <martin.noergaard@nru.dk>", "Anthony Galassi <28850131+bendhouseart@users.noreply.github.com>"]
+authors = ["Martin Nørgaard <martin.noergaard@nru.dk>", "Anthony Galassi <28850131+bendhouseart@users.noreply.github.com>", "Murat Bilgel <bilgelm@gmail.com>"]
 license = "MIT"
 readme = "README.md"
-include = [
-    "petdeface/*",
-    "pyproject.toml",
-]
-
-# please update the bids version to the latest compliant version when making modifications to this code here
-[tool.bids]
-bids_version = "1.8.0"
 
 [tool.poetry.dependencies]
-python = ">=3.8"
-scipy = "<1.9.2"
-nibabel = "^5.1.0"
-nipype = "^1.8.6"
-pybids = "^0.16.1"
-niworkflows = "^1.7.9"
-bids= ">=0.0.0"
-setuptools = "^67.7.2"
```

Comment:
Also, what happened to this include and the rest of the requirements?

Reply:
They are all requirements of niworkflows, so they are implicitly included.

Reply:
Include is required for packaging and pushing to pypi, see [here](https://python-poetry.org/docs/pyproject/#include-and-exclude). And is setup tools included too? So long as it builds and packages itself when your run `poetry build` and `poetry publish`.